client/src/components/OverLay/OverLay.css

```diff
@@ -0,0 +1,24 @@
+.overlay{
```

Comment:
This is not a proper overlay but background changed to grey, Overlay applies to the whole page (as it's only a background colour change there is no change to the line under theBank & login text & its line, grey line in nav is bright too).

Reply:
> Thanks for the review but can you share a link to a resource that will make me do the overlay properly

@mufidat3250 I made a PR #40 that should fix it if you follow that as a guide. The main issue was that the `Overlay` component was not being used. Instead the `background-color` was being set for the `Layout` component which was confusing and didn't properly apply the overlay color to the entire page. I refactored the overlay CSS and added the Overlay component to the relevant sections in the login and signup pages.

Let me know if you have any questions.

Reply:
I should probably mention, I made the `Overlay` component work such that, whenever it is used, it applies an overlay to the entire page _except_ for the child elements inside the `<Overlay>` and `</Overlay>` tags.

I think that's fairly intuitive, but it might make sense to rename the `Overlay` component to something like `ModalWithOverlay` or something like that. Feel free to refactor it in any way you feel makes more sense @mufidat3250.